.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ version: 2.1
 executors:
   machine_image:
     machine:
-      image: ubuntu-1604:201903-01
+      image: ubuntu-1604:202007-01
```

Comment:
@keitwb I think the test failures are related to https://github.com/kubernetes/kubernetes/issues/94281.  Updating to the latest machine image (from docker 18.09 to 19.03) resolves the failures.

Reply:
Looks like the metrics

```
blkio.io_service_bytes_recursive.read',
E         -            'blkio.io_service_bytes_recursive.write',
```
Are missing in the new docker version.  Just make the tests that look for them ignore them by removing them from the asserted set in pytest.